tests/test_optimizely.py

```diff
@@ -1436,7 +1436,7 @@ def test_get_feature_variable_boolean(self):
 
     mock_logger.assert_called_once_with(
       enums.LogLevels.INFO,
-      'Value for variable "is_working" of feature flag "test_feature_in_experiment" is true for user "test_user".'
+      'Value for variable "is_working" for variation "variation" is "true".'
```

Comment:
* feature_flag removed as we don't have feature key reference in the new method where we have moved this log message. Additionally, in logging mode logs mentioning feature keys precede this log so no need of it. 
* user_id same, not available. And the message sort of implied that selection of value somehow depends on the user ID which isn't true.